packages/cfd/src/Constants/cfd-password-modal-content/cfd-password-modal-strings.ts

```diff
@@ -0,0 +1,93 @@
+import { Jurisdiction, CFD_PLATFORMS } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import {
+    TDerivezCompanies,
+    TDxCompanies,
+    TMtCompanies,
+    getDerivezCompanies,
+    getDxCompanies,
+    getMtCompanies,
+} from '../../Stores/Modules/CFD/Helpers/cfd-config';
+
+type TGetTypeLabel = {
+    platform: string;
+    is_eu: boolean;
+    category: string;
+    type: string;
+};
+
+type TCancelButtonProps = {
+    should_set_trading_password: boolean;
+    error_type?: string;
+    is_mobile: boolean;
+};
+
+// TODO: Update with other platform and CFDs
+export const getWalletCFDTitle = (type: string) => {
+    switch (type) {
```

Comment:
@aizad-deriv merge it with the  `packages/cfd/src/Constants/cfd-password-modal-content/cfd-password-modal-content.ts`